lib/compiler/src/beam_ssa.erl

```diff
@@ -652,33 +653,27 @@ is_commutative('=/=') -> true;
 is_commutative('/=') -> true;
 is_commutative(_) -> false.
 
-def_used_1([#b_blk{is=Is,last=Last}|Bs], Preds, Def0, UsedAcc) ->
-    {Def,Used} = def_used_is(Is, Preds, Def0, used(Last)),
-    case Used of
-        [] ->
-            def_used_1(Bs, Preds, Def, UsedAcc);
-        [_|_] ->
-            def_used_1(Bs, Preds, Def, [Used|UsedAcc])
-    end;
-def_used_1([], _Preds, Def0, UsedAcc) ->
-    Def = ordsets:from_list(Def0),
-    Used = umerge(UsedAcc),
-    {Def,Used}.
+def_unused_1([#b_blk{is=Is,last=Last}|Bs], Preds, Def0, Used0) ->
```

Comment:
Shouldn't `Used` be called `Unused` at this point?

Reply:
Yes, it should! I have fixed it and force pushed.